openpype/hosts/maya/api/plugin.py

```diff
@@ -286,6 +291,8 @@ class MayaAutoCreator(AutoCreator, MayaCreatorBase):
         any arguments.
     """
 
+    settings_name = None
```

Comment:
Can be removed because it should be on `MayaCreatorBase` right?

Reply:
Its not on `MayaCreatorBase` though?